packages/web-components/src/components/footer/footer-container.ts

```diff
@@ -107,6 +107,7 @@ export function mapStateToProps(state: FooterContainerState): FooterContainerSta
     langDisplay,
     localeList: !language ? undefined : localeLists?.[language],
     links: !language ? undefined : translations?.[language]?.footerMenu,
+    footerLinks: !language ? undefined : translations?.[language]?.footerMenu,
```

Comment:
We need to fix this in `<dds-dotcom-shell-container>`, at: https://github.com/carbon-design-system/ibm-dotcom-library/blob/release/v1.10.0/packages/web-components/src/components/dotcom-shell/dotcom-shell-container.ts#L46

Reply:
Hmm ok, perhaps a follow-up PR after merging?